app/init.php

```diff
@@ -1,19 +1,31 @@
 <?php
+// Composer autoload
 require 'vendor/autoload.php';
+
+// Initialize Dotenv library
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+// initialize Twig template engine
 $loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
 $twigConfig = [];
-if (getenv('PRODUCTION')) {
+
+// If we're in production env, use cache
+if ($_ENV['PRODUCTION'] === true) {
     $twigConfig['cache'] = __DIR__ . '/cache';
+} else {
+    // if not production, use debug extension
+    $twigConfig['debug'] = true;
 }
+
 $twig = new \Twig\Environment($loader, $twigConfig);
+$twig->addExtension(new \Twig\Extension\DebugExtension());
 
-/*
-include_once $_ENV['BASE_DIRECTORY'] . '/web-assets/app_nav/header.php';
-include_once $_ENV['BASE_DIRECTORY'] . '/web-assets/app_nav/footer.php';
-include_once $_ENV['BASE_DIRECTORY'] . '/web-assets/app_nav/nav.php';
-*/
+// start a session
 session_start();
-?>
\ No newline at end of file
+
+// add our Dotenv variables to the Twig 'ENV' variable
+$twig->addGlobal('ENV', $_ENV);
```

Comment:
Once everything it set up, we have to explicitly tell twig what our `ENV` and `SESSION` variables are. With this, they will be available in templates as `{{ SESSION.user.name }}` etc.